pkg/generated/restapi/configure_rekor_server.go

```diff
@@ -200,7 +200,11 @@ func cacheForever(handler http.Handler) http.Handler {
 }
 
 func logAndServeError(w http.ResponseWriter, r *http.Request, err error) {
-	log.RequestIDLogger(r).Error(err)
+	if apiErr, ok := err.(errors.Error); ok && apiErr.Code() == http.StatusNotFound {
+		log.RequestIDLogger(r).Warn(err)
+	} else {
+		log.RequestIDLogger(r).Error(err)
```

Comment:
do you want to collapse the debug statement below into the else clause?

otherwise LGTM

Reply:
I left it that way on purpose, debug logs seem fine all the time, especially if they get more info out of the request.